client/src/Molecules.elm

```diff
@@ -120,7 +181,16 @@ initBackbone =
     , location = 0
     , bsa1Overhang = Nothing
     , bsmb1Overhang = Nothing
-    , sequence = ""
+    , sequenceLength = 0
+    , bacterialStrain = Nothing
+    , responsible = "" -- Owner of the Level 0 element = User that adds the vector.
```

Comment:
```suggestion
    , responsible = "" -- TODO: Owner of the Level 0 element = User that adds the vector.
```

Reply:
The TODO tags will be inserted again in the next commit.